lti_emailer/settings/base.py

```diff
@@ -124,11 +123,19 @@
         'OPTIONS': {
             'PARSER_CLASS': 'redis.connection.HiredisParser'
         },
-        'KEY_PREFIX': 'lti_emailer',  # Provide a unique value for shared cache
+        'KEY_PREFIX': 'lti_emailer',  # Provide a unique value for intra-app cache
         # See following for default timeout (5 minutes as of 1.7):
         # https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-CACHES-TIMEOUT
         'TIMEOUT': SECURE_SETTINGS.get('default_cache_timeout_secs', 300),
     },
+    'shared': {
+        'BACKEND': 'redis_cache.RedisCache',
+        'LOCATION': "redis://%s:%s/0" % (REDIS_HOST, REDIS_PORT),
```

Comment:
I'd recommend either having the shared cache use a common `KEY_PREFIX` that is used in any project that needs this functionality so that we can safely remove the shared keys without blowing away the entire cache.  Alternatively, we could have the shared cache live on a separate Redis DB instance (i.e. /1), but I think the prefix approach would work better.